src/BinSkim.Sdk/MetadataConditions.cs

```diff
@@ -40,5 +40,6 @@ public static class MetadataConditions
         public static readonly string ImageIsPreVersion7WindowsCEBinary = SdkResources.MetadataCondition_ImageIsPreVersion7WindowsCEBinary;
         public static readonly string ImageIsNativeUniversalWindowsPlatformBinary = SdkResources.MetadataCondition_ImageIsNativeUniversalWindowsPlatformBinary;
         public static readonly string MachOIsNotExecutableDynamicLibraryOrObject = SdkResources.MetadataCondition_MachOIsNotExecutableDynamicLibraryOrObject;
+        public static readonly string ImageIsARM64BitBinary = SdkResources.MetadataCondition_ImageIsARM64BitBinary;
```

Comment:
>ImageIsARM64BitBinary [](http://example.com/codeflow?start=38&length=21)

Casing of this field is 'ImageIsArm64BitBinary.

Also, you may observe that all these fields are sorted by line length, so please move this line to the appropriate spot (somewhere near 'ElfIsCoreNoneOrObject', I think).